OpenRA.Mods.Common/Traits/Buildable.cs

```diff
@@ -37,6 +37,12 @@ public class BuildableInfo : TraitInfo<Buildable>
 		[Desc("Palette used for the production icon.")]
 		[PaletteReference] public readonly string IconPalette = "chrome";
 
+		[Desc("Base build time (-1 indicates to use the unit's Value)")]
+		public readonly int BuildDuration = -1;
+
+		[Desc("Percentage modifier to apply to the build duration")]
+		public readonly int BuildDurationMultiplier = 150;
```

Comment:
Can you double check the naming conventions we use in other cases? IIRC these should be Modifier not Multiplier (because they dont multiply)